app/models/concerns/acts_as_taggable_on.rb

```diff
@@ -0,0 +1,72 @@
+module ActAsTaggableOn
+  def acts_as_taggable_on
+    class_eval do
+      def self.tagging_relation_name
+        "#{name.underscore}_tags".to_sym
+      end
+
+      has_many tagging_relation_name, :dependent => :destroy
+      has_many :tags, :through => tagging_relation_name
+
+      def self.taggable?
+        true
+      end
+
+      def taggings
+        # TODO: legacy v0.0 API dependency, remove when v0.0 is removed
```

Comment:
Does this comment mean that v1.0 shouldn't support what this method is returning? Or return a different payload? Or did Topology never make the switch when they moved to v1.0 ( I think they don't offer anything but v1.0. )

Reply:
I'm not sure on that one, I just copied this file straight from topological-inventory-core. I think it has something to do with how they used to model the tags/tagging but they've since changed it. This was there for backwards compatibility.